egs/formosa/s5/run.sh

```diff
@@ -0,0 +1,435 @@
+#!/bin/bash
+#
+# Copyright 2018, Yuan-Fu Liao, National Taipei University of Technology, yfliao@mail.ntut.edu.tw
+#
+# Instruction
+#
+# Before you run this recips, please apply, download and put or make a link of the corpus under this folder (folder name: "NER-Trs-Vol1"). for more detail, please check:
+# 1. Formosa Speech in the Wild (FSW) project (https://sites.google.com/speech.ntut.edu.tw/fsw/home/corpus)
+# 2. Formosa Speech Recognition Challenge (FSW) 2018 (https://sites.google.com/speech.ntut.edu.tw/fsw/home/challenge)
+#
+# Then, you could excute "run.sh" to train models and decode test data. There are many switchs in this sceript (flag "true" or "false"). you could set them to "false" to bypass centain steps. ), for example:
+#
+# false && (
```

Comment:
Please remove any old comments, like this one,reduce the amount of vertical whitespace, and instead of the "true && ... " mechanism that you are using, make this script accept the "--stage" option which you can use to continue after a partial rerun.  You'll need to source utils/parse_options.sh.

Reply:
Thanks! All old comments, the "true && ... " mechanism and vertical whitespace were removed now.